CHANGELOG.md

```diff
@@ -11,6 +11,9 @@ and this project adheres to [Semantic Versioning](http://semver.org/).
 
 - New `InterruptNumber` trait is now required on interrupt arguments to the
   various NVIC functions, replacing the previous use of `Nr` from bare-metal.
+- Associated const `PTR` is introduced to Core Peripherals to
+  replace the existing `ptr()` API. `ptr()` API is to be removed in the
```

Comment:
I think the plan was to mark as deprecated in 0.7 and remove it for good in 1.0? Anyway I'd just leave it out for now, we can always tweak the deprecation/removal plan in a separate PR.